app/src/main/java/org/glucosio/android/tools/GlucoseRanges.java

```diff
@@ -53,56 +61,57 @@ void setPreferredRange(String preferredRange) {
 
     @VisibleForTesting
     void setCustomMin(int customMin) {
-        this.customMin = customMin;
+        this.userMin = customMin;
     }
 
     @VisibleForTesting
     void setCustomMax(int customMax) {
-        this.customMax = customMax;
+        this.userMax = customMax;
+    }
+
+    public static int getPresetMin(String preset) {
+        switch (preset) {
+            case ADA:
```

Comment:
Super minor: You can combine with `default` case

Reply:
Perhaps that is a better solution to the question below. Remove one of the cases and use default instead?